dbt_common/exceptions/__init__.py

```diff
@@ -0,0 +1,7 @@
+from dbt_common.exceptions.base import *  # noqa
```

Comment:
Minor nit, but `mypy` can be configured at the repo level to skip checking for unused imports on `__init__.py` files. We put this everywhere, but could make it more explicit (and a bit visually cleaner) if we put it in the config.